packages/components/src/DataList/hooks/useActiveLayout.ts

```diff
@@ -0,0 +1,15 @@
+import { useResponsiveSizing } from "./useResponsiveSizing";
+import { useDataListContext } from "../context/DataListContext";
+
+export function useActiveLayout() {
+  const { layoutBreakpoints, layouts } = useDataListContext();
+  const breakpoints = useResponsiveSizing();
+  const reversedLayoutBreakpoints = [...layoutBreakpoints].reverse();
```

Comment:
Why did you revert it?

Reply:
So it checks the highest breakpoint first and see if it's true